client/signup/jetpack-connect/plan-features-tab/index.jsx

```diff
@@ -0,0 +1,378 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import PlanFeatures from 'my-sites/plan-features';
+import SegmentedControl from 'components/segmented-control';
+import SegmentedControlItem from 'components/segmented-control/item';
+import {
+	PLAN_JETPACK_FREE,
+	PLAN_JETPACK_PREMIUM,
+	PLAN_JETPACK_BUSINESS,
+	PLAN_JETPACK_PERSONAL,
+	PLAN_JETPACK_PREMIUM_MONTHLY,
+	PLAN_JETPACK_BUSINESS_MONTHLY,
+	PLAN_JETPACK_PERSONAL_MONTHLY,
+} from 'lib/plans/constants';
+import FAQ from 'components/faq';
+import FAQItem from 'components/faq/faq-item';
+import purchasesPaths from 'me/purchases/paths';
+
+const jetpackPlansPersonalTab = [ PLAN_JETPACK_FREE, PLAN_JETPACK_PERSONAL ];
+const jetpackPlansPremiumTab = [ PLAN_JETPACK_PREMIUM, PLAN_JETPACK_BUSINESS ];
+const jetpackMonthlyPlansPersonalTab = [ PLAN_JETPACK_FREE, PLAN_JETPACK_PERSONAL_MONTHLY ];
+const jetpackMonthlyPlansPremiumTab = [ PLAN_JETPACK_PREMIUM_MONTHLY, PLAN_JETPACK_BUSINESS_MONTHLY ];
+
+class PlansFeaturesMain extends Component {
+	constructor( props ) {
+		super( props );
+		this.state = { currentTab: 'personal' };
+		this.changePlanGroup = this.changePlanGroup.bind( this );
+	}
+
+	changePlanGroup() {
+		this.state.currentTab =
+			this.setState( {
+				currentTab: this.state.currentTab === 'personal'
+					? 'premium'
+					: 'personal'
+			}
+		);
+	}
+
+	renderPlanTypeSelector() {
+		const { translate } = this.props;
+
+		return (
+			<div>
+				<SegmentedControl primary className="plan-features-tab__selector">
+					<SegmentedControlItem
+						selected={ this.state.currentTab === 'personal' }
+						onClick={ this.changePlanGroup }>
+							{ translate( 'Personal' ) }
+					</SegmentedControlItem>
+					<SegmentedControlItem
+						selected={ this.state.currentTab === 'premium' }
+						onClick={ this.changePlanGroup }>
+							{ translate( 'Business' ) }
+					</SegmentedControlItem>
+				</SegmentedControl>
+			</div>
+		);
+	}
+
+	isJetpackSite( site ) {
+		return site.jetpack;
+	}
+
+	getCurrentViewablePlans() {
+		let plans;
+		if ( this.props.intervalType === 'monthly' ) {
+			plans = this.state.currentTab === 'personal'
```

Comment:
This might be a nitpick but maybe we can use constants for the tabs. Something like `TAB_PERSONAL` and `TAB_PREMIUM`